singularity-opac/neutrinos/thermal_distributions_neutrinos.hpp

```diff
@@ -27,7 +27,7 @@ namespace neutrinos {
 
 using pc = PhysicalConstants<CGS>;
 
-#define EPS (10.0 * std::numeric_limits<Real>::epsilon())
+#define EPS (10.0 * std::numeric_limits<Real>::min())
```

Comment:
constexpr?

Reply:
Maybe `EPS` should be defined in `neutrinos/base` or something, so we don't have the same line floating around everywhere.

Reply:
As above I don't see a clear path forward for shared `EPS` at the moment due to C++14